src/trackHttp.test.js

```diff
@@ -0,0 +1,85 @@
+import {trackPromise, emitter} from './trackHttp';
+import {Emitter} from './tinyemmiter';
+
+
+describe('trackPromise', () => {
+  test('On Initial case, promise fired, promise emitter.emit is called', () => {
```

Comment:
I understand that the idea of this unit test is to check that .emit() is called once we start tracking a given promise, and it should not be called again until that promise is resolved/rejected. In light of this, I would rename it to something like "when tracking a single, still pending promise, emitter.emit should be called once"